tests/ml_commons/test_ml_commons_client.py

```diff
@@ -372,6 +372,26 @@ def test_integration_model_train_register_full_cycle():
                     print("Model Task Status:", ml_task_status)
                     raised = True
                 assert raised == False, "Raised Exception in pulling task info"
+
+                try:
+                    search_task_obj = ml_client.search_task(json="{'_id':'{task_id}'}")
```

Comment:
I saw parameter name was: `json_input`. Then how can we invoke with `json`?

Reply:
Yes, my bad sorry. I will change it.